src/engine_test.go

```diff
@@ -41,3 +43,81 @@ func TestCanWriteRPrompt(t *testing.T) {
 		assert.Equal(t, tc.Expected, got, tc.Case)
 	}
 }
+
+func BenchmarkEngineRender(b *testing.B) {
```

Comment:
isn't this more like a main test -> or easier to setup when you call main with args?

Reply:
This is more like the main test. But I did not want to do the trick where the test binary calls itself with additional arguments, so I just copied most of the setup code into the test. Making it a dedicated main test is going to be better, but I am not sure how exactly to do that.

Reply:
me neither, but we could try to find out :-)

Reply:
Let's keep it here for the time being, I'll try to refactor that later.